<launch>
   <node name="roboclaw_python" pkg="roboclaw_python" type="roboclaw_node.py" output="screen" >
		<param name="port" value="/dev/roboclaw" />
		<param name="baud_rate" value="1000000" />
		<param name="update_rate" value="50" />
		<param name="base_width" value="0.35" />
		<param name="ticks_per_metre" value="19660.16" />
		<param name="base_frame_id" value="base_link" />
		<param name="robot_direction" value="1" />
		<param name="max_acceleration" value="0.5" />
		<param name="QPPS" value="12000" />
   </node>
</launch>
